Content.Shared/Teleportation/Components/TeleportLocationsComponent.cs

```diff
@@ -45,6 +45,12 @@ public sealed partial class TeleportLocationsComponent : Component
     /// </summary>
     [DataField]
     public LocId? Speech;
+
+    /// <summary>
+    /// Maximum radius around the beacon the user could teleport to.
+    /// </summary>
+    [DataField("maxRandomRadius"), ViewVariables(VVAccess.ReadWrite)]
```

Comment:
```suggestion
    [DataField]
```
Redundant. Your IDE should show a warning for this.